plugins/bulk-import/src/components/PreviewFile/PreviewFileSidebarDrawerContent.tsx

```diff
@@ -59,7 +59,7 @@ const useDrawerContentStyles = makeStyles(theme => ({
     bottom: 0,
     paddingLeft: '24px',
     paddingBottom: '24px',
-    backgroundColor: theme.palette.background.paper,
+    backgroundColor: theme.palette.type === 'light' ? '#fff' : '#292929',
```

Comment:
I was expecting the drawer footer to be of the same as the color of the footer in the main form, but it isn't

![Screenshot 2024-09-13 at 9 10 27 PM](https://github.com/user-attachments/assets/de269ca2-ce00-42e2-8d79-18d734dbf2aa)

Reply:
Hi @debsmita1 , the footer background color should match the main content background color where the footer is placed. @ShiranHi has confirmed this color design.